tests/_files/GeneratorTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
```

Comment:
Is this file still needed? (if yes, then what about a more descriptive name?)

Reply:
Probably good to have one integration test for the originally reported issue. Agree the name was not descriptive and changed it